### PR TITLE
Skeleton setup

### PR DESCRIPTION
**Pull request by parkerpersaud**
@andarcabrera @aaronlahey

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# tictactoe-java
+# tictactoe-java [![Build Status](https://travis-ci.org/klkelley/tictactoe-java.svg?branch=skeleton-setup)](https://travis-ci.org/klkelley/tictactoe-java)
```
Comment (aaronlahey) [inline README.md:1]: Nice. I'd also like to see instructions here on how to do common tasks like run the tests, build the project, run the project, etc.

File: pom.xml [hunk 2]
```diff
@@ -7,6 +7,17 @@
     <groupId>8thlight</groupId>
     <artifactId>tictactoe</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>RELEASE</version>
```
Comment (aaronlahey) [inline pom.xml]: I would specify an explicit version number here.

File: pom.xml [hunk 3]
```diff
@@ -7,6 +7,17 @@
...
+        </dependency>
+    </dependencies>
+    <properties>
+        <maven.compiler.source>1.6</maven.compiler.source>
```
Comment (aaronlahey) [inline pom.xml]: As it's specified here, you'll only be able to use features from Java 6 and earlier. If you change these to 1.8 you'll get to use those features.

File: src/main/java/Main.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+public class Main {
```
Comment (aaronlahey) [inline src/main/java/Main.java]: Typically Java projects don't use classes without package. A lot of times the packages correspond to domains the person/company owns. What about `me.karakelley.tictactoe`.

File: src/test/java/MainTest.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import org.junit.Test;
```
Comment (aaronlahey) [inline src/test/java/MainTest.java]: Same comment here about packages.

File: tictactoe.iml [hunk 6]
```diff
@@ -11,5 +11,7 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Maven: junit:junit:4.12" level="project" />
```
Comment (aaronlahey) [inline tictactoe.iml]: You should let Maven handle dependency resolution. I think IntelliJ has functionality to "import" Maven projects. Are you using that? That would let you make updates to your pom.xml and IntelliJ will know where to find everything it needs to find.

File: pom.xml [hunk 7]
```diff
@@ -7,6 +7,17 @@
     <groupId>8thlight</groupId>
```
Comment (aaronlahey) [inline pom.xml]: I would change this `groupId` to `me.karakelley` to match your package structure.

**Review by aaronlahey — CHANGES_REQUESTED** [review]

File: tictactoe.iml (hunk 6 above)
Comment (parkerpersaud) [reply tictactoe.iml]: I opted-in to auto-import of maven projects - I'm not sure if that is the same thing you are referring to. After I opted-in nothing was changed in my .iml file however.

Comment (aaronlahey) [reply tictactoe.iml]: Ok. If you are using auto-import then that's perfect! If you change the dependencies in your pom.xml IntelliJ might update this file to keep it in sync. I'm not too sure what IntelliJ does when it "auto-imports" stuff.

Nothing to change, then.

File: README.md [hunk 8]
```diff
@@ -1 +1,13 @@
...
+
+
+### Build & Run
+```
+$ mvn clean install
+$ mvn exec:java
```
Comment (aaronlahey) [inline README.md]: I'd like to be able to run the project without `mvn`. So look into how to build a jar file, and give me instructions on how to do that (using `mvn`) and then run that jar file. Once that's done I'm cool with this being merged.

**Pull request merged**